src/python/gem5/utils/multisim/multisim.py

```diff
@@ -181,8 +187,11 @@ def _run(module_path: Path, id: str) -> None:
     except Exception as e:
         inform(f"Error running simulator {id}: {e}")
 
+    pipe.send(sim_list[0].get_stats())
+    pipe.close()
+
 
-def run(module_path: Path, processes: Optional[int] = None) -> None:
+def run(module_path: Path, processes: Optional[int] = None) -> Dict[str, Dict]:
     """Run the simulators specified in the module in parallel.
```

Comment:
It'd be good to update the documentation string here to explain what this function returns